src/components/vm/vmMigrateDialog.jsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState } from 'react';
+import {
+    Alert,
+    Button,
+    Checkbox,
+    Form,
+    FormGroup,
+    Modal,
+    Radio,
+    TextInput,
+    Tooltip
+} from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
+
+import { migrateToUri } from '../../libvirt-dbus.js';
+import { isEmpty, isObjectEmpty } from '../../helpers.js';
+import { ModalError } from 'cockpit-components-inline-notification.jsx';
+
+const _ = cockpit.gettext;
+
+const DestUriRow = ({ validationFailed, destUri, setDestUri }) => {
+    return (
+        <FormGroup label={_("Destination URI")} fieldId="dest-uri"
+                   id="dest-uri"
+                   validated={validationFailed.destUri ? "error" : "default"}>
+            <TextInput id='dest-uri-input'
+                       validated={validationFailed.destUri ? "error" : "default"}
+                       value={destUri}
+                       placeholder={cockpit.format(_("Example, $0"), "qemu+ssh://192.0.2.16/system")}
+                       onChange={setDestUri} />
+        </FormGroup>
+    );
+};
+
+const OptionsRow = ({ temporary, live, setTemporary, setLive, setStorage }) => {
+    return (
+        <FormGroup label={_("Options")}
+                   fieldId="options"
```

Comment:
Same here, fieldId should be the id of the nested element. in this case you have a group, so you can add an aria-label instead.